sdk/storage/Azure.Storage.Files/src/DirectoryClient.cs

```diff
@@ -676,15 +678,13 @@ private void SetNameFieldsIfNull()
                 Pipeline.LogMethodEnter(
                     nameof(DirectoryClient),
                     message:
-                    $"{nameof(Uri)}: {Uri}\n" +
-                    $"{nameof(shareSnapshot)}: {shareSnapshot}");
+                    $"{nameof(Uri)}: {Uri}");
                 try
                 {
                     Response<RawStorageDirectoryProperties> response = await FileRestClient.Directory.GetPropertiesAsync(
                         ClientDiagnostics,
                         Pipeline,
                         Uri,
-                        sharesnapshot: shareSnapshot,
```

Comment:
Shouldn't this line remain, and be changed to ```sharesnapshot: this.Snapshot```?

Reply:
No, the snapshot is passed along in the `Uri`.  There is no `this.Snapshot` to pass.